cli/src/pcluster/resources/compute_node/user_data.sh

```diff
@@ -35,6 +35,10 @@ export NO_PROXY="localhost,127.0.0.1,169.254.169.254"
 PROXY
 fi
 
+if [ "${OSUser}" == "rocky" ]; then
```

Comment:
Do we have to check the operating systems?
Can we just remove the `ALL=(ALL) NOPASSWD:ALL` from `/etc/sudoers` on any OSes?
Same question for other `user_data.sh`

Reply:
Moreover, can we make this action only when `disable_sudo_access_default_user` is true?

Reply:
> Moreover, can we make this action only when disable_sudo_access_default_user is true?
Will do
> Do we have to check the operating systems?
None of the other OS has that policy its just rocky

Reply:
What if a custom AMI is used?

Reply:
If the base Ami has the policy then it would be required to be removed. Maybe removing the OS check would be okay as the command would work only if the policy string doesn’t exist